README.md

```diff
@@ -5,6 +5,7 @@
 | [![Build Status](https://github.com/BOINC/boinc/workflows/Android/badge.svg)](https://github.com/BOINC/boinc/actions?query=workflow:Android+branch:master) | [![codecov](https://codecov.io/gh/BOINC/boinc/branch/master/graph/badge.svg)](https://codecov.io/gh/BOINC/boinc) |
 | [![Build Status](https://github.com/BOINC/boinc/workflows/Linux/badge.svg)](https://github.com/BOINC/boinc/actions?query=workflow:Linux+branch:master) | [![Coverity Scan Build Status](https://scan.coverity.com/projects/4226/badge.svg)](https://scan.coverity.com/projects/boinc-boinc) |
 | [![Build Status](https://github.com/BOINC/boinc/workflows/Linux-MinGW/badge.svg)](https://github.com/BOINC/boinc/actions?query=workflow:Linux-MinGW+branch:master) | [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/BOINC/boinc/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/BOINC/boinc/?branch=master) |
+| [![Build Status](https://github.com/BOINC/boinc/actions/workflows/snap.yml/badge.svg?branch=master)](https://github.com/BOINC/boinc/actions/workflows/snap.yml) | [![Test Report](https://github.com/BOINC/boinc/actions/workflows/test-report.yml/badge.svg?branch=master)](https://github.com/BOINC/boinc/actions/workflows/test-report.yml)|
```

Comment:
Please remove `?branch=master` to let GitHub use the default branch

Reply:
ok. All the other is using the master branch. So we should fix it for all.

Reply:
done.